ssh/tunnel.py

```diff
@@ -16,8 +16,6 @@
 
 from pkgpanda.util import write_string
 
-LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
```

Comment:
If there is code which is invoking these modules `tunnel.py`, `test_util/aws.py` **outside** of dcos-launch, then we are effectively loosing the formatting now. Isn't it?  

Is it good idea to have a common formatted logger?